### PR TITLE
[INLONG-2170] add Inlong-Sort-standalone document.

### PR DESCRIPTION
**Pull request by luchunliang**
Fixes apache/incubator-inlong#2170

where *XYZ* should be replaced by the actual issue number.

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: docs/modules/sort-standalone/overview.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+### supported data storage
+- Apache hive (currently only supports sequence file format)
+- Apache Pulsar
+- Apache Kafka
+
+### Future planning
+#### support more types of cache layer message queues
+Apache Kafka, etc.
+
```
Comment (dockerzhang) [inline docs/modules/sort-standalone/overview.md:43]: too many empty lines.

File: docs/modules/sort-standalone/quick_start.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+The installation file is located in the 'inlong sort standard / sort standard dist / target /' directory. The file name is Apache inlong sort standard - ${project. Version} - bin tar. gz。
+
+## Start the inlong sort standalone application
+With the tar produced in the above compilation stage After you unpack the GZ package, you can start the inlong sort standalone application.
+Example:
+
+```
+./bin/sort-start. sh
+```
+
```
Comment (dockerzhang) [inline docs/modules/sort-standalone/quick_start.md:16]: too many empty lines.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/sort-standalone/overview.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+### 支持的数据存储
+- Apache Hive（当前只支持sequence文件格式）
+- Apache Pulsar
+- Apache Kafka
+
+### 未来规划
+#### 支持更多种类的缓存层消息队列
+Apache Kafka等。
+
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/sort-standalone/overview.md:37]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/sort-standalone/overview.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+#### 支持更多种类的数据存储
+Hbase，ElasticSearch等。
+
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/sort-standalone/overview.md:41]: ditto

File: docs/modules/sort-standalone/quick_start.md (hunk 2 above)
Comment (luchunliang) [reply docs/modules/sort-standalone/quick_start.md:14]: fixed

File: docs/modules/sort-standalone/overview.md (hunk 1 above)
Comment (luchunliang) [reply docs/modules/sort-standalone/overview.md:37]: fixed

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/sort-standalone/overview.md (hunk 3 above)
Comment (luchunliang) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/sort-standalone/overview.md:37]: fixed

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/sort-standalone/overview.md (hunk 4 above)
Comment (luchunliang) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/sort-standalone/overview.md:40]: fixed

**Review by gosonzhang — APPROVED** [review]
+1

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**